packages/libs/components/src/map/styles/map-styles.css

```diff
@@ -121,3 +121,16 @@
   box-shadow: -1px -0.5px 0px 6px rgba(255, 255, 0, 1),
     -1px -0.5px 0px 2px rgba(255, 255, 0, 1) inset;
 }
+
+/* area selection for selecting markers */
+.leaflet-area-selection-box {
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 0;
+  height: 0;
+  box-sizing: border-box;
+  z-index: 800;
+  border: 2px dotted rgba(255, 255, 0, 1);
```

Comment:
I think it looks pretty cool with the same styling as the marker highlighting

```suggestion
  border: 6px solid rgba(255, 255, 0, 1);
```

Reply:
![image](https://github.com/VEuPathDB/web-monorepo/assets/308639/37c18ece-4c89-442f-bd81-5325b3ac96ff)

Reply:
@bobular Thank you for your suggestion 👍 Then I will comply with your suggestion and merge this into original PR